.github/workflows/pr.yml

```
@@ -13,6 +14,10 @@ jobs:
       - uses: wagoid/commitlint-github-action@v5
       - name: Check unpinned versions
         run: ./.github/scripts/pin-version
+      - name: Check TypeScript types
+        run: yarn install
```

suggestion: you should ensure the lockfile is not altered
```suggestion
        run: yarn install --frozen-lockfile
```
Same suggestion if you have any other instances of `yarn install` in the CI (or local development instructions in a README)